File: 16challenge/main.cpp
#include <iostream>
using namespace std;

int main() {
    int cigarettesPerDay, yearsSmoked;
    cout &#8203;`oaicite:{"index":0,"invalid_reason":"Malformed citation << \"Enter the number of cigarettes smoked per day: \";\n    cin >>"}`&#8203; cigarettesPerDay;
    cout &#8203;`oaicite:{"index":1,"invalid_reason":"Malformed citation << \"Enter the number of years smoked: \";\n    cin >>"}`&#8203; yearsSmoked;

    int minutesLostPerCigarette = 10;
    int minutesLostPerDay = cigarettesPerDay * minutesLostPerCigarette;
    int minutesLostPerYear = minutesLostPerDay * 365;
    int minutesLostTotal = minutesLostPerYear * yearsSmoked;
    int daysLost = minutesLostTotal / (24 * 60);

    cout << "The smoker will lose " << daysLost << " days of life due to smoking." << endl;

    return 0;
}

File: 16challenge/main.py
cigarettes_per_day = int(input("Enter the number of cigarettes smoked per day: "))
years_smoked = int(input("Enter the number of years smoked: "))

minutes_lost_per_cigarette = 10
days_lost = sum(cigarettes_per_day * minutes_lost_per_cigarette for year in range(years_smoked)) / (24 * 60)

print("The smoker will lose {} days of life due to smoking.".format(days_lost))
